lib/cb/clients/recommendation.rb

```diff
@@ -69,7 +69,9 @@ def self.for_company(company_did)
         if json_hash
           api_jobs = json_hash.fetch('Results', {}).fetch('JobRecommendation', {}).fetch('Jobs', {})
           if api_jobs
-            api_jobs.fetch('CompanyJob', []).each do |cur_job|
+            company_jobs = api_jobs.fetch('CompanyJob', [])
+            company_jobs = [company_jobs] unless company_jobs.is_a?(Array)
+            company_jobs.each do |cur_job|
```

Comment:
So, there's another edge case that this doesn't handle: what if we fetch `'CompanyJob'` and it returns `nil` like `'Jobs'` did? There is some nifty ruby that you could do instead though!

```ruby
company_jobs = [api_jobs.fetch('CompanyJob', [])].flatten.compact
company_jobs.each ...
```

The `flatten` will ensure your array is only 1 level deep. The `compact` will remove any `nil`s.